extras/log_file_client/logs/index.html

```diff
@@ -1,10 +1,18 @@
 <html>
-    <body>
-        <ul>
-            <li><a href="/logs/ProjectA-tank-24.log">/logs/ProjectA-tank-24.log</a></li>
-            <li><a href="/logs/ProjectA-tank-70.log">/logs/ProjectA-tank-70.log</a></li>
-            <li><a href="/logs/ProjectB-tank-58.log">/logs/ProjectB-tank-58.log</a></li>
-            <li><a href="/logs/index.html">/logs/index.html</a></li>
-        </ul>
-    </body>
+
+<head>
+    <title>Index of /logs/</title>
+</head>
+
+<body>
+    <h1>Index of /logs/</h1>
+    <hr>
+    <pre>
+        <a href="ProjectA-tank-24.log">ProjectA-tank-24.log</a>
+        <a href="ProjectA-tank-70.log">ProjectA-tank-70.log</a>
+        <a href="ProjectB-tank-58.log">ProjectB-tank-58.log</a>
+    </pre>
+    <hr>
+</body>
+
 </html>
```

Comment:
The red circle with a line indicates that there is not a newline at the end of the file. We try to include this.